examples/local/vstream_client.go

```diff
@@ -47,7 +47,7 @@ func main() {
 				Shard:    "-80",
 				// Gtid "" is to stream from the start, "current" is to stream from the current gtid
 				// you can also specify a gtid to start with.
-				Gtid: "", //"current"  // "MySQL56/36a89abd-978f-11eb-b312-04ed332e05c2:1-265"
+				Gtid: "", //"current"  // "MySQL/36a89abd-978f-11eb-b312-04ed332e05c2:1-265"
```

Comment:
My suggestion is as follows:
- Don't change GTID format. That will break contracts with clients
- Feel free to change function names etc.